src/lib/utils/form.utils.ts

```
@@ -4,3 +4,9 @@ export function preventDefault(fn: (event: Event) => any) {
 		fn.call(this, event);
 	};
 }
+
+// Helper function to get input values from the DOM
+export function getInputValue(id: string, defaultValue: string = ''): string {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)